plugins/env/lib/samson_env/samson_plugin.rb

```diff
@@ -38,12 +38,17 @@ def write_dotenv_file(base_file, groups)
 
     # writes a proprietary .json file with a env hash for each deploy group
     def write_env_json_file(env_json, manifest_json, groups)
-      return unless File.exist?(env_json)
-      json = JSON.load(File.read(env_json))
-      File.unlink(env_json)
+      return unless File.exist?(manifest_json)
```

Comment:
not a big fan of this gem ... lots of code for very little functionality + it's private ...